### ISSUE TITLE
Traceback error when using Icetea with Python 3

### ISSUE THREAD

**Issue by MarceloSalazar** [ARMmbed/icetea#37]
### Description

Evaluating 5.10 during OOB round 2.
Found Traceback error

Environment:

- Mbed CLI 1.8.0
- Python 3.7.0
- Both: Windows 10 and OSX El Capitan

```
>icetea --version
Traceback (most recent call last):
  File "C:\Users\marsal01\AppData\Local\Programs\Python\Python37-32\Scripts\icetea-script.py", line 11, in <module>
    load_entry_point('icetea==1.0.1', 'console_scripts', 'icetea')()
  File "c:\users\marsal01\appdata\local\programs\python\python37-32\lib\site-packages\icetea_lib\main.py", line 20, in icetea_main
    from icetea_lib import IceteaManager
  File "c:\users\marsal01\appdata\local\programs\python\python37-32\lib\site-packages\icetea_lib\IceteaManager.py", line 35, in <module>
    from icetea_lib.TestSuite.TestSuite import TestSuite, SuiteException
  File "c:\users\marsal01\appdata\local\programs\python\python37-32\lib\site-packages\icetea_lib\TestSuite\TestSuite.py", line 33, in <module>
    from icetea_lib.TestSuite.TestcaseContainer import TestStatus, DummyContainer
  File "c:\users\marsal01\appdata\local\programs\python\python37-32\lib\site-packages\icetea_lib\TestSuite\TestcaseContainer.py", line 29, in <module>
    from icetea_lib.bench import ReturnCodes
  File "c:\users\marsal01\appdata\local\programs\python\python37-32\lib\site-packages\icetea_lib\bench.py", line 53, in <module>
    from icetea_lib.DeviceConnectors.Dut import DutConnectionError
  File "c:\users\marsal01\appdata\local\programs\python\python37-32\lib\site-packages\icetea_lib\DeviceConnectors\Dut.py", line 330
    self.execute_command(cli_cmd[0], wait=wait, async=asynchronous)
                                                    ^
SyntaxError: invalid syntax
```

[ ] Question  
[ ] Enhancement  
[X] Bug  

Mirroring issue from: https://github.com/ARMmbed/mbed-os/issues/8116
ARM Internal Ref: MBOTRIAGE-1648

**Comment by jupe** [comment]
@jonikula we have to ensure first that our CI can detect this issue and then provide fix. On fact there shouldn’t be anymore any of those python reserved names..

**Comment by jonikula** [comment]
Fix in PR #40

**Comment by jupe** [comment]
fix will be included in next release
